force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -6152,6 +6152,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Close button label</shortDescription>
         <value>Close</value>
     </labels>
+    <labels>
+        <fullName>commonCancelAndClose</fullName>
+        <categories>Button</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cancel and close label</shortDescription>
+        <value>Cancel and close</value>
```

Comment:
Should this be "Cancel and Close"?
Has this label been reviewed by docs?

Reply:
yes, this corresponds to the modal's close icon, as defined in the slds documentation